pennylane/__init__.py

```diff
@@ -33,6 +33,7 @@
 import pennylane.templates
 import pennylane.hf
 import pennylane.qchem
+import pennylane.quantum_info
```

Comment:
I'm wondering if it's better to shorten the module name to `pennylane.qinfo`, both for conciseness and to keep with the `qchem` naming standard.

Reply:
I like it too, changed!